<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgHooksBase</class>
 <widget class="QWidget" name="DlgHooksBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Hook settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_urlArgument">
     <property name="title">
      <string>URL Argument</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Custom commands on incoming calls with URL</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_protocols" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QCheckBox" name="kcfg_enableHooksSIP">
           <property name="text">
            <string>Trigger on specific SIP header</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KLineEdit" name="kcfg_hooksSIPHeader">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_enableHooksIAX">
        <property name="text">
         <string>Trigger on IAX2 URL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_urlArgumentForm" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="label_command">
           <property name="text">
            <string>Command to run</string>
           </property>
           <property name="buddy">
            <cstring>kcfg_hooksCommand</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KLineEdit" name="kcfg_hooksCommand">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>%s will be replaced with the passed URL.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_phoneNumberFormatting">
     <property name="title">
      <string>Phone Number Rewriting</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <item>
       <widget class="QWidget" name="widget_phoneNumberFormattingForm" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QCheckBox" name="kcfg_addPrefix">
           <property name="text">
            <string>Prefix dialed numbers with</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KLineEdit" name="kcfg_prepend">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_configHooks">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>499</width>
       <height>96</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_enableHooksSIP</sender>
   <signal>toggled(bool)</signal>
   <receiver>DlgHooksBase</receiver>
   <slot>setChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_enableHooksIAX</sender>
   <signal>toggled(bool)</signal>
   <receiver>DlgHooksBase</receiver>
   <slot>setChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>-2</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_addPrefix</sender>
   <signal>toggled(bool)</signal>
   <receiver>DlgHooksBase</receiver>
   <slot>setChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_hooksCommand</sender>
   <signal>textEdited(QString)</signal>
   <receiver>DlgHooksBase</receiver>
   <slot>setChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_hooksSIPHeader</sender>
   <signal>textEdited(QString)</signal>
   <receiver>DlgHooksBase</receiver>
   <slot>setChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>518</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_prepend</sender>
   <signal>textEdited(QString)</signal>
   <receiver>DlgHooksBase</receiver>
   <slot>setChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setChanged()</slot>
 </slots>
</ui>
